clean-lumber.lic

```diff
@@ -35,23 +35,23 @@ class CleanLumber
         end
         size = right_hand.split.last
       end
-      unless bput('get my saw', 'wood saw', 'You get') == 'wood saw'
-        fput('stow saw')
+      unless /woodcutting saw|wood saw/ =~ bput('get my saw', 'wood saw', 'woodcutting saw', 'You get')
+        fput("put my saw in my #{args.storage}")
```

Comment:
Is it always the case that the container in which you store lumber should be the same container in which you store your tools?